🚧 Provide proposal of KEDA governance (#663)
kedacore/keda: GOVERNANCE.md

[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
the other project maintainers agrees to extend the period to be greater than 3
months.

New maintainers can be added to the project by a
[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
vote of the existing maintainers. A potential maintainer may be nominated by an
existing maintainer. A vote is conducted in private between the current
maintainers over the course of a one week voting period. At the end of the week,
votes are counted and a pull request is made on the repo adding the new
maintainer to the [MAINTAINERS](MAINTAINERS.md) file.

A maintainer may step down by submitting an
[issue](https://github.com/kedacore/keda/issues/new) stating their intent.

Changes to this governance document require a pull request with approval from a
[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
the current maintainers.

## Managing Projects

New projects can be added to the organization after a successful
[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
vote of the existing maintainers.

> Contributors who want to donate a scaler, sample or other contribution must

Reviewer: Makes sense for scalers, I'm just wondering if the vote process for contributions in general is scalable.

Author: It's more talking about contributing new repos in our org, what about this:
```suggestion
Contributors who want to donate an add-on scaler, sample or donate a repo must
```
